chromium/rules.js

```diff
@@ -16,13 +16,15 @@ const trivial_cookie_name_c = new RegExp(".*");
 const trivial_cookie_host_c = new RegExp(".*");
 
 // Empty iterable singleton to reduce memory usage (#12494).
-const nullIterable = Object.create(null, {
-  [Symbol.iterator]: {
-    value: function* () {
-      // do nothing
-    }
+const nullIterator = {
+  next : () => { done: true }
+};
+
+const nullIterable = {
+  [Symbol.iterator]: function *() {
```

Comment:
what?

Reply:
`*`

Reply:
Simply change it into `() => nullIterator` I guess.

Reply:
without `*` it does not work for me.

Reply:
@cschanaj See https://github.com/EFForg/https-everywhere/pull/12577/files#r139175667.

Reply:
Remove the * after fixing this issue.

Reply:
Remove the * now.